dj_hetmech_app/views.py

```diff
@@ -72,10 +79,18 @@ def get(self, request):
 
 
 class QueryMetapathsView(APIView):
+    """
+    Return metapaths between a given source and target node whose path count
+    information is stored in the database. The database only stores a single
+    orientation of a metapath. For example, if GpPpGaD is stored between the
+    given source and target node, DaGpPpG would not also be stored. Therefore,
+    both orientations of a metapath are searched against the PathCount table.
+    """
     http_method_names = ['get']
 
     def polish_pathcounts(self, source_node, target_node, pathcounts_data):
-        """This function polishes pathcounts_data. The polishment includes:
+        """
```

Comment:
Are you sure this is what you want? It is against PEP8:
https://www.python.org/dev/peps/pep-0008/#documentation-strings

Reply:
This is not against PEP8 as described in https://github.com/python/black/issues/144#issuecomment-387872212. I agree with the many commenters there that putting the openning `"""` on the same line as text is a bit of an eyesore.

Reply:
Okay with me if you prefer that.